CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
+## [0.0.next] - 2017-06-02
+### Changed
+- java-heap-pcnt.sh and metrics-java-heap-graphite.sh: removed extra column from 'UsedHeap'
```

Comment:
Doing a bit of refactoring based on java versions. based on the following:
* It looks like there is some redundant repetition in the scripts in there.
* heap includes PermGen prior to Java 8 and Java 8 has metaspace that is not considered to be part of the heap. It'd be good to have someone more knowledgeable about java memory confirm that.
* no point calling jstat multiple times
* the number of columns that jstat returns based on version:

** Java 8. 17 columns:
`root@vagrant vagrant]# /opt/gls/jdk8/current/bin/jstat -gc 3254
 S0C    S1C    S0U    S1U      EC       EU        OC         OU       MC     MU    CCSC   CCSU   YGC     YGCT    FGC    FGCT     GCT   
14848.0 14336.0  0.0   812.8  146944.0 146939.6  79360.0    19773.3   36696.0 36037.2 4736.0 4456.7      9    0.092   2      0.147    0.238`

** Java 7. 15 columns:
`[root@qalocal vagrant]# /usr/lib/jvm/java-1.7.0-openjdk-1.7.0.141.x86_64/bin/jstat -gc 3384
 S0C    S1C    S0U    S1U      EC       EU        OC         OU       PC     PU    YGC     YGCT    FGC    FGCT     GCT   
4224.0 4224.0  0.0    0.0   34112.0   3428.4   503040.0   100359.8  262144.0 86647.1     96    1.277  24      7.391    8.668`


Java 6. 15 columns:
`[root@vagrant vagrant]# /opt/gls/jdk6/jdk1.6.0_45/bin/jstat -gc 3384
 S0C    S1C    S0U    S1U      EC       EU        OC         OU       PC     PU    YGC     YGCT    FGC    FGCT     GCT   
4224.0 4224.0  0.0    0.0   34112.0   3052.3   503040.0   100359.8  262144.0 86647.1     96    1.277  24      7.391    8.668`